starter/script.js

```diff
@@ -1 +1,131 @@
-// put code here!
+var username = prompt("Hello there! You've been given a chance to go back in time to meet with some of modern history's most important political and popular figures. Before you meet them, you'll need to answer a few questions, and do some preperation for your meeting. Are you all set? What is your name");
+
+var year = prompt("Hello, " + username +  " Okay, let's select our destination. We have a few options. Enter the decade you'd like to visit: (1950/2030)")
+    if (year === "1950") {
+      var year = prompt("Heading to the 50s! Interesting time. Would you like to have lunch with President Harry Truman or Frank Sinatra? (T/S)")
+    } else if (year === "2030") {
+      prompt("Just kidding! This hasn't happened yet! Why not try 1950?")
+  } while (year != "1950" || "2030") {
+  	prompt("Please select a valid choice!")
```

Comment:
Good idea, forcing them to make a valid choice with the `while` loop, but you're never capturing their input! You need to explicitly capture the return value from prompt via assignment or it will be lost.

```js
year = prompt("Please select a valid choice!")
```